common/djangoapps/student/urls.py

```diff
@@ -3,65 +3,58 @@
 """
 
 from django.conf import settings
-from django.conf.urls import patterns, url
+from django.conf.urls import url
+from django.contrib.auth.views import password_reset_complete
 
-from student.views import LogoutView
-
-urlpatterns = (
-    'student.views',
+from student.views import *
 
+urlpatterns = [
     url(r'^logout$', LogoutView.as_view(), name='logout'),
 
     # TODO: standardize login
 
     # login endpoint used by cms.
-    url(r'^login_post$', 'login_user', name='login_post'),
+    url(r'^login_post$', login_user, name='login_post'),
     # login endpoints used by lms.
-    url(r'^login_ajax$', 'login_user', name="login"),
-    url(r'^login_ajax/(?P<error>[^/]*)$', 'login_user'),
+    url(r'^login_ajax$', login_user, name="login"),
+    url(r'^login_ajax/(?P<error>[^/]*)$', login_user),
 
-    url(r'^email_confirm/(?P<key>[^/]*)$', 'confirm_email_change'),
+    url(r'^email_confirm/(?P<key>[^/]*)$', confirm_email_change),
 
-    url(r'^create_account$', 'create_account', name='create_account'),
-    url(r'^activate/(?P<key>[^/]*)$', 'activate_account', name="activate"),
+    url(r'^create_account$', create_account, name='create_account'),
+    url(r'^activate/(?P<key>[^/]*)$', activate_account, name="activate"),
 
-    url(r'^accounts/disable_account_ajax$', 'disable_account_ajax', name="disable_account_ajax"),
-    url(r'^accounts/manage_user_standing', 'manage_user_standing', name='manage_user_standing'),
+    url(r'^accounts/disable_account_ajax$', disable_account_ajax, name="disable_account_ajax"),
+    url(r'^accounts/manage_user_standing', manage_user_standing, name='manage_user_standing'),
 
-    url(r'^change_setting$', 'change_setting', name='change_setting'),
-    url(r'^change_email_settings$', 'change_email_settings', name='change_email_settings'),
+    url(r'^change_setting$', change_setting, name='change_setting'),
+    url(r'^change_email_settings$', change_email_settings, name='change_email_settings'),
 
     # password reset in student.views (see below for password reset django views)
-    url(r'^password_reset/$', 'password_reset', name='password_reset'),
+    url(r'^password_reset/$', password_reset, name='password_reset'),
     url(
         r'^password_reset_confirm/(?P<uidb36>[0-9A-Za-z]+)-(?P<token>.+)/$',
-        'password_reset_confirm_wrapper',
+        password_reset_confirm_wrapper,
```

Comment:
This one is still referenced by module path in a `{% url %}` tag in `lms/templates/registration/password_reset_email.html`

Reply:
Should be fixed now